other/cmr-exchange/service-bridge/test/cmr/opendap/tests/integration/app/ous.clj

```diff
@@ -12,13 +12,64 @@
   Definition used for integration tests:
   * https://en.wikipedia.org/wiki/Software_testing#Integration_testing"
   (:require
+   [cheshire.core :as json]
    [clojure.test :refer :all]
+   [cmr.http.kit.request :as request]
    [cmr.http.kit.response :as response]
    [cmr.opendap.testing.system :as test-system]
+   [cmr.opendap.testing.util :as util]
    [org.httpkit.client :as httpc]))
 
 (use-fixtures :once test-system/with-system)
 
+(deftest cmr-hits-and-search-after-test
+  (let [collection-id "C1200442179-HMR_TME"
+        options (-> {}
+                    (request/add-token-header (util/get-sit-token)))]
+    (testing "cmr-hits no page size with 2 hits"
+      (let [response @(httpc/get
+                       (format (str "http://localhost:%s"
+                                    "/service-bridge/ous/collection/%s")
+                               (test-system/http-port)
+                               collection-id)
+                       options)]
+        (is (= 200 (:status response)))
+        (is (= "12" (get-in response [:headers :cmr-hits])))
+        (is (= "[\"hmr_tme\",1031097600000,1200442190]" (get-in response [:headers :cmr-search-after])))))
+    (testing "cmr-hits page size 1 with 2 hits"
+      (let [response @(httpc/get
+                       (format (str "http://localhost:%s"
```

Comment:
The base url can be constructed at line 27 and used in the tests rather than constructing it every single time from scratch. Also I don't see the point of changing the `page_size` in the search-after tests.